src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -1,61 +1,78 @@
-import React, { useState } from 'react';
-import { View, Alert } from 'react-native';
-import { TextInput } from 'react-native-paper';
-import login from '../../util/login';
-import { Title, LinkButton, SpacerInline } from '../../elements';
-import InputLabel from '../../elements/FormTextInput/InputLabel';
-import styles from './LoginScreen.styles';
-import { useNavigation, useNavigationParam } from 'react-navigation-hooks';
+import React, { useState } from "react";
+import { View, Alert, Button } from "react-native";
+import { TextInput } from "react-native-paper";
+import login from "../../util/login";
+import { Title, LinkButton, SpacerInline } from "../../elements";
+import InputLabel from "../../elements/FormTextInput/InputLabel";
+import styles from "./LoginScreen.styles";
+import { useNavigation, useNavigationParam } from "react-navigation-hooks";
 
 export default () => {
-	const { navigate } = useNavigation();
-	const [ email, setEmail ] = useState(useNavigationParam('email') || '');
-	const [ password, setPassword ] = useState(useNavigationParam('password') || '')
+  const { navigate } = useNavigation();
+  const [email, setEmail] = useState(useNavigationParam("email") || "");
+  const [password, setPassword] = useState(
```

Comment:
Why the formatting change on this one but not the others?

Reply:
My fault, I used an extension on VS code which automatically change the quotes and this line. I've already changed them back